suggest/elasticsearch/src/main/java/com/spotify/heroic/suggest/elasticsearch/SuggestBackendKV.java

```diff
@@ -578,7 +581,19 @@ public boolean isReady() {
                         .setId(suggestId)
                         .setSource(suggest)
                         .setOpType(DocWriteRequest.OpType.CREATE)
-                        .execute()).directTransform(response -> timer.end()).onDone(writeContext));
+                        .execute())
+                        .directTransform(response -> timer.end())
+                        .catchFailed(throwable -> {
+                            if (throwable instanceof VersionConflictEngineException) {
+                                /* Index request rejected, document already exists. That's ok,
+                                 * return success. */
+                                reporter.reportWriteDroppedByDuplicate();
+                                return WriteSuggest.of();
+                            }
+                            log.error("Suggest write failed: ", throwable);
+                            throw new RuntimeException(throwable);
+                        })
```

Comment:
Pull this to a helper method?

```
catchFailed(this::handleVersionConflict)
```

Logging should not be necessary. If it's needed, it should happen downstream since the exception is propagated.